runtime/gc_glue_java/MarkingDelegate.cpp

```diff
@@ -130,6 +130,7 @@ MM_MarkingDelegate::workerSetupForGC(MM_EnvironmentBase *env)
 {
 	GC_Environment *gcEnv = env->getGCEnvironment();
 	gcEnv->_markJavaStats.clear();
+	gcEnv->_continuationStats.clear();
```

Comment:
minor, but let's move the clear() (and merge()) methods, to be after those for mark and scavenge stats, so to match the order of calls to the order of structs definitions